CMakeLists.txt

```diff
@@ -104,23 +104,25 @@ IF (WIN32)
 	# winmm.lib must come before kernel (or older version of 32-bit windows
 	# will have linking issues for certain entry points)
 	SET(JUCE_PLATFORM_SPECIFIC_LIBRARIES
+                winmm.lib
+                ws2_32.lib
+                wininet.lib
+                version.lib
+                Shlwapi.dll
+		)
+	SET(JUCE_APPARENTLY_NOT_NEEDED_LIBRARIES
```

Comment:
Once you cut out `JUCE_APPARENTLY_NOT_NEEDED_LIBRARIES`, let's merge this

Reply:
Done, merge away!

I also updated the comment about ordering, and added a comment above the mystery `-DJUCE_DONT_AUTOLINK_TO_WIN32_LIBRARIES`, which [further reading](http://jucebrary.sourceforge.net/docs/doxygen/juce__core_8h.html#afc230a49a0164d4264431704bbb0d947) has revealed is a Visual C++-specific thang.